src/nuxeo-cloud/day-to-day/support.md

```diff
@@ -0,0 +1,74 @@
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 500
+toc: true
+---
+
+## Support
+
+Nuxeo is available to help support issues that come up after the environments have been provisioned and start being used. Typical issues include:
+- Support for application or performance issues
+- Accidental deletion of Data
+- Data restoration from Prod to Pre-Prod
+- Deploy additional marketplace packages, 3rd party services, or configuration updates
+
+The Nuxeo support team provides first level support to evaluate all issues and route to the appropriate teams, including NCO.
+
+Nuxeo is available 24x7 to support site availability issues. If other issues are found, the team will address them following our standard customer support SLA’s depending on subscription level.
+
+**Jira** is the primary method to notify us of any issues:
+- Hours of Support: only during normal business hours (9AM - 5PM EST)
+- Additional support may be provided if resources are available and we are notified of critical project timelines/milestones
+
+In the event issues are encountered that cannot be resolved, we can provide support to roll back the environment to the initial settings.
+
+### Opening a support ticket on Cloud Environments
+
+When [filling a JIRA Ticket]({{page version='' space='studio' page='how-to-fill-a-jira-ticket'}}), there are some important aspects:
+- Nuxeo logs and config are not required for investigations with Nuxeo Cloud environments, however screenshots and code may be.
+- You don't need to specify the packages and server version deployed on your instance
+
+## Maintenance
+
+### Maintenance Windows
+
+Our standard maintenance window is 7 days a week:
+- EU customers: 12AM - 2AM CET
+- US customers: 12AM - 2AM EST
+
+Exceptions can be made to accommodate peak user times.
+
+### Production and Pre-Production Maintenance
+
+All Nuxeo Cloud Production and Pre-Production environments are supported via standard maintenance windows. These windows are available to the Nuxeo Cloud team 7 days a week in order to make any necessary updates to environments. They are typically used only 1-2 times per month.
+
+## Hotfixes, Patches, 3rd Party Updates and LTS releases
+
+The Cloud team will proactively deploy and manage the following monthly, during the first week of each month.
+- Nuxeo Hotfixes
+- Server Patches
+- 3rd party service updates
+- Infrastructure updates
+
+The LTS upgrade process will be coordinated with customers and the appropriate development team/professional services to identify risks and minimize impact
+- LTS releases are supported up to 3 years after the release has been made public.
+- A non-compliance fee may be imposed for customers who do not upgrade to the new LTS after an LTS release is no longer supported.
+
+## Nuxeo Hotfixes
+
+Nuxeo Cloud applies Nuxeo LTS Hotfixes to all cloud environments the first full week of each month during the maintenance windows.
```

Comment:
change to 

Nuxeo Cloud applies Nuxeo LTS Hotfixes to all cloud environments the first and second week of each month during the maintenance window. Nuxeo Cloud customer contacts are emailed 1 week before the Hotfix deployment takes place.

Our standard deployment is:

Pre-Prod: First Thursday of the Month
Prod: Second Thursday of the Month